Automattic/wp-calypso
# Add data e2e selectors

No functional changes.

Add data attributes for e2e tests - as a part of removing `xpath` selectors from e2e tests https://github.com/Automattic/wp-e2e-tests/issues/30

Testing:
- View comments for some blog post and make sure that comment `comments__comment-content` has additional `data-e2e-comment` selector. If comment text is _'Test Comment'_, selector should be `data-e2e-comment="Test Comment"`
- Open notification panel and make sure that comment `wpnc__excerpt` has additional `data-e2e-comment` selector. Selector should be structured same as above.
